answer/answer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"$schema": "http://json-schema.org/draft/2019-09/schema#",
-	"$id": "https://github.com/dedis/student20_pop/tree/proto-specs/answer/answer.json",
+	"$id": "https://raw.githubusercontent.com/dedis/student20_pop/proto-specs/answer/answer.json",
```

Comment:
Can you explain why this change was needed?

Reply:
I'm not sure that this is needed, but it does make sense. That way the $id can actually be resolved if some library tries to fetch it, as it points to the file itself rather than Github interface.

Reply:
The library we use in fe2 has the abitily to get the latest json directly by downloading it. So it makes more sense that this url points to a valid json file.

This is not needed

Reply:
I'd still consider that this is a good change to implement.

HOWEVER, careful with that sort of functionality in fe2, as if the protocol evolves but not your application, you end up with something broken, and the links are not versioned. Your application shouldn't be relying on un-versioned external dependencies that may change at any point. That's why most REST API endpoints (for instance) are designed to remain stable through time and versioned, so that even if they do change, the old version remains available with that exact same specification.

You could rely on the online thing IFF you start creating folders for versions, i.e.
`"$id": "https://raw.githubusercontent.com/dedis/student20_pop/proto-specs/1.0/answer/answer.json",`
but I wouldn't go there ;-)